### PR TITLE
[IMP][FIX] crm_lead_product: fix indentation ..

### PR DESCRIPTION
**Pull request by Rad0van**
.. not to update lead's revenue after each line

### CODE REVIEW

**Review by flotho — APPROVED** [review]
easy,
LGTM
Thanks

File: crm_lead_product/models/crm_lead.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def _onchange_lead_line_ids(self):
         for lead_line in self.lead_line_ids:
             if lead_line.planned_revenue != 0:
                 planned_revenue += lead_line.planned_revenue
-                self.planned_revenue = planned_revenue
+            self.planned_revenue = planned_revenue
```
Comment (rousseldenis) [inline crm_lead_product/models/crm_lead.py]: Nearly. One more indentation back @Rad0van :sweat_smile:

Comment (rousseldenis) [inline crm_lead_product/models/crm_lead.py]:
```suggestion
        self.planned_revenue = planned_revenue
```

**Comment by rafaelbn** [comment]
Hello @Rad0van ! you have to suggestions from @rousseldenis . Could you please attend them please? Thank you! 😄

**Review by rafaelbn — APPROVED** [review]
Please FIX indentation @Rad0van !

Thanks 👍

**Comment by rafaelbn** [comment]
@rousseldenis you final review! All done!

**Comment by Rad0van** [comment]
@rafaelbn @flotho @rousseldenis can you guys please merge this as it should be OK to go? Thank you.

**Comment by rafaelbn** [comment]
Could you make an empty commit to chech if runboat awake! ? 😄

**Pull request closed without merging**